metricbeat/module/kubernetes/apiserver/_meta/fields.yml

```diff
@@ -36,8 +36,8 @@
       type: long
       description: >
         Request latency, number of requests
-    - name: request.latency.bucket
```

Comment:
This should not change the template itself as it checks if a `*` already exists in the name and if not it will add it. We use the `*` in the docs check to verify that the fields are multifields. We should use this as a convention moving forward.